website/package-lock.json

```diff
@@ -52,7 +52,7 @@
     },
     "../dist/clr-addons": {
       "name": "@porscheinformatik/clr-addons",
-      "version": "19.2.0",
+      "version": "19.2.1",
```

Comment:
wrong version here. you need to build "npm run build" the main project, to update the version there and then once again execute "npm i" in website folder